File: LeetCode/Array/0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int zero = 0;
        int n = nums.length;
        for(int i = 0 ; i < n ; i++){
            if(nums[i] != 0){
                nums[zero] = nums[i];
                zero++;
            }

        }
        while(zero < n){
            nums[zero] = 0;
            zero++;
        }
    }
}

File: LeetCode/Array/0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
 
        nonzero = 0
        n = len(nums)
        for i in range(n):
            if nums[i] != 0:
                nums[nonzero] = nums[i]
                nonzero+=1

        while(nonzero < n):
            nums[nonzero] = 0
            nonzero+=1
        